views/js/controller/runner/runner.js

```diff
@@ -102,43 +72,41 @@ define([
         /**
          * Controller entry point
          *
-         * @param {Object}   options - the testRunner options
-         * @param {String}   options.testDefinition - the test definition id
-         * @param {String}   options.testCompilation - the test compilation id
-         * @param {String}   options.serviceCallId - the service call id
-         * @param {Object}   options.bootstrap - contains the extension and the controller to call
-         * @param {String}   options.exitUrl - the full URL where to return at the final end of the test
-         * @param {Object[]} options.plugins - the collection of plugins to load
-         * @param {Object[]} options.providers - the collection of providers to load
+         * @param {Object} config - the testRunner config
+         * @param {String} config.testDefinition - the test definition id
+         * @param {String} config.testCompilation - the test compilation id
+         * @param {String} config.serviceCallId - the service call id
+         * @param {Object} config.bootstrap - contains the extension and the controller to call
+         * @param {Object} config.options - the full URL where to return at the final end of the test
+         * @param {Object[]} config.providers - the collection of providers to load
          */
-        start: function start(options) {
-            var runnerOptions = _.defaults({}, options, defaults);
-            var exitReason;
-            var $container = $('.runner');
-            var logger = loggerFactory('controller/runner', {
-                serviceCallId : runnerOptions.serviceCallId,
-                plugins : Object.keys(runnerOptions.plugins)
+        start(config) {
+            let exitReason;
+            const $container = $('.runner');
+            const logger = loggerFactory('controller/runner', {
+                serviceCallId : config.serviceCallId,
+                plugins : Object.keys(config.providers.plugins)
```

Comment:
This line cause a javascript error:
`Uncaught TypeError: Cannot convert undefined or null to object`
I think it should be: `plugins : Object.keys(config.providers)`

Reply:
The error is between the chair and the keyboard... I've again forgotten to link one of the PR, if the server sends the correct values, it works. However I'll move this logger after the validation